iosApp/iosApp/Pages/Favorites/FavoritesView.swift

```diff
@@ -34,7 +36,8 @@ struct FavoritesView: View {
                             string: NSLocalizedString("Edit", comment: "Button text to enter edit favorites flow"),
                             backgroundColor: Color.translucentContrast,
                             textColor: Color.fill2,
-                            height: 32
+                            height: editButtonHeight,
+                            width: 64
```

Comment:
Scaling width made the button too wide at the largest text sizes, such that the page title of "Favorites" was cut off.